src/server/engine_shard_set.cc

```diff
@@ -30,6 +33,13 @@ ABSL_FLAG(bool, cache_mode, false,
           "If true, the backend behaves like a cache, "
           "by evicting entries when getting close to maxmemory limit");
 
+ABSL_FLAG(uint64_t, mem_defrag_threshold, 20,
```

Comment:
Add flag to control the threshold to trigger the operation.
I took 20 as the ratio between memory for the application and committed memory. Do I need to change that?
I use 1.3 as the ration between used memory and committed memory - does it make sense?

Reply:
it's better to use doubles, and maybe set 0.1 as default. In any case, add the range to docs string so it will be clear which values are acceptable.